### DISCUSSION TITLE
Is there a list of suggested gestures or shortcuts?

### DISCUSSION THREAD

**Discussion by ano327** [rxhanson/Multitouch-Community#213] [Q&A]
I recall using this app years ago but since have lost all my settings. I'd love it if there were a quick hit list of really great gestures I can just import. Is there a file or list of these or video demo of these that people love, that I can see?

**Comment by rxhanson** [comment]
Thanks for returning! It's kind of weird to say this, but because I don't gather user data I don't actually have a set of most commonly used gestures/actions. In all the exposure that I've had to configuration from users, the app is typically customized really differently per user. It's been a while since I've thought about this, but maybe it would make sense to finally build out a place to share configuration. I still prefer not gathering user data, but might consider adding an opt-in way to send analytics.

Personally, I like the rest four/five, remove all but one trackpad gestures the best :)
